java/ql/src/experimental/Security/CWE/CWE-598/SensitiveGetQuery.ql

```diff
@@ -0,0 +1,58 @@
+/**
+ * @name Sensitive GET Query
+ * @description Use of GET request method with sensitive query strings.
+ * @kind path-problem
+ * @id java/sensitive-query-with-get
+ * @tags security
+ *       external/cwe-598
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+import semmle.code.java.frameworks.Servlets
+import semmle.code.java.security.SensitiveActions
+import DataFlow::PathGraph
+
+/** Finds variables that hold sensitive information judging by their names. */
+class SensitiveInfoExpr extends Expr {
+  SensitiveInfoExpr() {
+    exists(Variable v | this = v.getAnAccess() |
+      v.getName().regexpMatch(getCommonSensitiveInfoRegex())
+    )
+  }
+}
+
+/** GET servlet method of `javax.servlet.http.Servlet` and subtypes. */
```

Comment:
```suggestion
/** Holds if `c` is a call to some override of `HttpServlet.doGet`. */
```

Reply:
Thanks @smowton for reviewing this PR. I've made the change.